styleguide/source/_patterns/01-molecules/08-lists/02-list-rule-metadata.twig

```diff
@@ -0,0 +1,23 @@
+{% include '07-list_title-small/list_title-small.twig' with { 'content': 'Downloads'} %}
+
+{# No metadata, list links rule #}
+
+<ul class="list_links list_links-rule">
+  <li class="list_links_item">
+    {% include '00-link.twig' with { 'content': 'Lorem ipsum dolor', 'class': 'link-black' } %}
+    {% include '10-list-link-meta.twig' with { 'content': 'PDF, 2MB' } %}
+  </li>
+  <li class="list_links_item">
+    {% include '00-link.twig' with { 'content': 'Donec non lectus leo', 'class': 'link-black' } %}
+    {% include '10-list-link-meta.twig' with { 'content': 'PDF, 5MB' } %}
+  </li>
+  <li class="list_links_item">
+    {% include '00-link.twig' with { 'content': 'Lorem ipsum dolor', 'class': 'link-black' } %}
+    {% include '10-list-link-meta.twig' with { 'content': 'PDF, 3MB' } %}
+  </li>
+  {# List Link item example with metadata #}
+  <li class="list_links_item">
+    {% include '00-link.twig' with { 'content': ' Donec non lectus leo', 'class': 'link-black' } %}
+    {% include '10-list-link-meta.twig' with { 'content': 'PDF, 2MB' } %}
```

Comment:
@aCyborg we ran into this with the Load More icon too. Do either of you know any more sensible way to write the twig includes so that including icons and such in links would be easier?